lib/ceo/rails/routes.rb

```diff
@@ -19,6 +19,8 @@ def admin_for(*rsrcs, &block)
       options[:concerns] = :pageable
 
       namespace :admin do
+        get 'dashboard', to: 'admin#dashboard'
```

Comment:
There has to be a better way to define this route other than recreating it every time the method is called. Any thoughts @WendyBeth @mattsears?